src/test/java/uk/gov/hmcts/dts/fact/controllers/admin/list/AdminAreasOfLawControllerTest.java

```diff
@@ -47,4 +44,135 @@ void shouldReturnAllAreasOfLaw() throws Exception {
             .andExpect(status().isOk())
             .andExpect(content().json(allAreasOfLawJson));
     }
+
+    @Test
+    void shouldReturnAnAreaOfLaw() throws Exception {
+        final AreaOfLaw areaOfLaw = getAreasOfLaw().get(0);
+        when(adminAreasOfLawService.getAreaOfLaw(100)).thenReturn(areaOfLaw);
+
+        final String areaOfLawJson = new ObjectMapper().writeValueAsString(areaOfLaw);
+
+        mockMvc.perform(get(BASE_PATH + "/100").accept(MediaType.APPLICATION_JSON))
+            .andExpect(status().isOk())
+            .andExpect(content().json(areaOfLawJson));
+    }
+
+    @Test
+    void shouldReturnNotFoundWhenAreaOfLawIdNotFound() throws Exception {
+        when(adminAreasOfLawService.getAreaOfLaw(400)).thenThrow(NotFoundException.class);
+
+        mockMvc.perform(get(BASE_PATH + "/400").accept(MediaType.APPLICATION_JSON))
+            .andExpect(status().isNotFound());
+    }
+
+    @Test
+    void updateAreaOfLawShouldUpdateAndReturnUpdatedAreaOfLaw() throws Exception {
+        final AreaOfLaw areaOfLaw = getAreasOfLaw().get(0);
+        areaOfLaw.setExternalLink("https://a.different.url");
```

Comment:
Don't think u need to set a new URL here. This test is using a mock value so regardless of what is the URL, it always return what you are expecting it to return as you use "when...thenReturn" in the test.